src/isWilsonPrime.js

```diff
@@ -20,6 +20,17 @@
  */
 function isWilsonPrime(num) {
   // write code here
+    let factNum = num - 1;
+    if (factNum === 0 || factNum === 1)
+      factNum = 1;
+    for (var i = factNum - 1; i >= 1; i--) {
+      factNum *= i;
+    }
+    if (Number.isInteger((factNum + 1) / (num * num)) === true) {
```

Comment:
this if have no sense.

you can write simple
```suggestion
   return Number.isInteger((factNum + 1) / (num * num));
```

Reply:
`Number.isInteger` - return Boolean value